GoMeet/src/test/java/com/google/sps/VoteMeetingTimeServletTest.java

```diff
@@ -109,7 +109,7 @@ public void testSuccessfulVoteTime() throws IOException {
       put("status", HttpServletResponse.SC_OK);
     }};
     writer.flush(); // writer may not have been flushed yet
-    assertTrue(stringWriter.toString() != null); 
+    assertNotNull(stringWriter.toString()); 
```

Comment:
For fix-ups like this, consider using ```git rebase -i``` and the fix-up option to combine this with the commit where the tests were introduced. I think that would make the commit history cleaner and if you have to find the commit where this block of code was introduced, it is in one place.

Reply:
Yeah, given that both cf4f0d8 and 21c0e2c are just minor fix-ups that don't really need to be recorded in their own commit, I've squashed them both into 5fafbcb, where the main functionality is 😊